src/maintainer/knowledge_base.rst

```diff
@@ -1518,3 +1518,27 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Migrators and Migrations
+========================
+
+When any changes are made in the global pinnings of a package, then the entire stack of the packages that depend on that package would need to be updated and rebuilt.
+Doing it manually could be quite tedious, and that's where migrations come to help. Migrations are used to change how different feedstocks/packages operate and are an integral part of ``regro-cf-autotick-bot``'s duties.
+
+To generate these migrations, you use migrators. Migrators automatically create pull requests for the affected packages in conda-forge. 
```

Comment:
There are several types of migrations. Most of the time we only need an updated pin, which can be triggered via migration files. However there are others. I'd mention the `cf-scripts` docs: https://regro.github.io/cf-scripts/migrators.html too